app/workers/pubsubhubbub/delivery_worker.rb

```diff
@@ -18,6 +18,7 @@ def perform(subscription_id, payload)
     return if DomainBlock.blocked?(host)
 
     headers['User-Agent']      = 'Mastodon/PubSubHubbub'
+    headers['Content-Type']    = 'application/x-www-form-urlencoded'
```

Comment:
Is this actually correct...?

This is supposed to inform a subscriber about what content we're sending them, so I think something like `application/atom+xml` would be expected?